qase-cucumberjs/src/reporter.ts

```diff
@@ -1,22 +1,32 @@
 import { EventEmitter } from 'events';
 
 import { Formatter, IFormatterOptions, Status } from '@cucumber/cucumber';
-import { Envelope, PickleTag, TestCaseStarted } from '@cucumber/messages';
+import {
+  Envelope,
+  PickleStep,
+  PickleTag,
+  TestCaseStarted,
+  TestStepFinished,
+} from '@cucumber/messages';
 
 import {
-  ConfigType,
+  composeOptions,
   ConfigLoader,
-  QaseReporter,
-  ReporterInterface,
+  ConfigType,
+  QaseReporter, Relation,
+  ReporterInterface, StepStatusEnum,
+  StepType,
   TestStatusEnum,
-  composeOptions,
+  TestStepType,
```

Comment:
Both StepType and TestStepType?

Reply:
yes. StepType is an enum. TestStepType is a model.